packages/jellyfish-transaction/__tests__/script/dftx/dftx_loans/setDefaultLoanScheme.test.ts

```diff
@@ -0,0 +1,57 @@
+import { SmartBuffer } from 'smart-buffer'
+import {
+  CSetDefaultLoanScheme,
+  SetDefaultLoanScheme
+} from '../../../../src/script/dftx/dftx_loans'
+import { OP_CODES } from '../../../../src/script'
+import { toBuffer, toOPCodes } from '../../../../src/script/_buffer'
+import { OP_DEFI_TX } from '../../../../src/script/dftx'
+
+it('should bi-directional buffer-object-buffer', () => {
+  const fixtures = [
+    '6a0d446654786407736368656d6532'
+  ]
+
+  fixtures.forEach(hex => {
+    const stack = toOPCodes(
+      SmartBuffer.fromBuffer(Buffer.from(hex, 'hex'))
+    )
+    const buffer = toBuffer(stack)
+    expect(buffer.toString('hex')).toStrictEqual(hex)
+    expect((stack[1] as OP_DEFI_TX).tx.type).toStrictEqual(0x64)
+  })
+})
+
+const header = '6a0d4466547864' // OP_RETURN(0x6a) (length 13 = 0x0d) CDfTx.SIGNATURE(0x44665478) CSetDefaultLoanScheme.OP_CODE(0x64)
+
```

Comment:
extra line ?

Reply:
Hahaha, ok I have removed the extra line. Also I apply the same changes to createLoanScheme code.